kani-compiler/src/args.rs

```diff
@@ -4,6 +4,20 @@
 use strum_macros::{AsRefStr, EnumString, VariantNames};
 use tracing_subscriber::filter::Directive;
 
+#[derive(Debug, Default, Clone, Copy, AsRefStr, EnumString, VariantNames, PartialEq, Eq)]
+#[strum(serialize_all = "snake_case")]
+pub enum BackendOption {
+    /// Aeneas (LLBC) backend
+    Aeneas,
+
+    /// CProver (Goto) backend
+    CProver,
+
+    /// Backend option was not explicitly set
+    #[default]
+    None,
```

Comment:
Why none? Shouldn't we make the argument optional instead?

Reply:
I think I was trying to simplify the backend checking logic under different features. I'll clean this up.